app/models/trade/annual_report_upload.rb

```diff
@@ -49,8 +49,8 @@ def validation_errors
     @primary_validation_errors || @secondary_validation_errors
   end
 
-  def sandbox
-    return nil if is_submitted?
+  def sandbox(tmp=false)
```

Comment:
Why do we need this param? Would be good to document what it does. I'm slightly worried about what happens when tmp is true and the report is submitted? I don't think we should ever proceed past this line if report is submitted, because the table no longer exists in such case and hell will ensue.